creator-onboarding/readme/minter-suite.md

```diff
@@ -0,0 +1,69 @@
+---
+order: 851
+---
+
+# Minter Suite
+
+This page provides an overview of the Art Blocks Minter Suite, which enables artists to choose how they distribute their artwork to collectors. on a per-project basis.
+
+The current available Minter options are discussed below. We are continually expanding our minter suite over time.
+
+!!!info
+For details on determining proper pricing and configuring your project's minter, please see the [Minter section in the Project Form Fields Guide](project-form-fields-guide.md#minter).
+!!!
+
+## `Set Price - ETH`
+
+The set price minter is used for fixed price releases. It is the simplest minter and prices all tokens at the same price, in ETH.
```

Comment:
Do we want to share recommendations here re: bot mitigation? Maybe that shouldn't live in the doc and it should be a conversation, but mentioning it here could also offer context before partners ask us when we're onboarding.

Reply:
I think this is a great callout. Now that this page will exist, I'm going to ping @sarahrossien and @madpinney and invite them to add whatever information they want with respect to bot mitigation, since they regularly consult artists in this area 🤝